en/docs/deploy/configure-user-store-preference-order.md

```diff
@@ -0,0 +1,41 @@
+# Configure the user store preference order
+
+From WSO2 Identity server 5.9.0 onwards, admins can configure the user store preference order for a service provider during the authentication.
+
+This guide helps you to configure the user store preference order for a service provider.
+
+## Initial set up
+
+To set up the WSO2 identity Server to configure the user store preference order:
+
+1. Implement the `UserStorePreferenceOrderSupplier` interface with your own logic to retrieve the user store order.
+
+    Use the provided template to implement the [UserStorePreferenceOrderSupplier interface](https://github.com/wso2/carbon-kernel/blob/feeae3e4668b805c8d6d5e8c115897fa93b8a856/core/org.wso2.carbon.user.core/src/main/java/org/wso2/carbon/user/core/config/UserStorePreferenceOrderSupplier.java?source=post_page-----cdadf43f9366--------------------------------).
```

Comment:
Let's validate these links. Seems there are some query parameters. Is it intentional?